implementations/rust/ockam/ockam_node/src/channel_types.rs

```diff
@@ -0,0 +1,31 @@
+/// Sender used to send payload messages
+pub type MessageSender<T> = tokio::sync::mpsc::Sender<T>;
+/// Receiver used to receive payload messages
+pub type MessageReceiver<T> = tokio::sync::mpsc::Receiver<T>;
+
+/// Create message channel
+pub fn message_channel<T>() -> (MessageSender<T>, MessageReceiver<T>) {
+    tokio::sync::mpsc::channel(16)
+}
+
+/// Router sender
+pub type RouterSender<T> = tokio::sync::mpsc::Sender<T>;
+/// Router receiver
+pub type RouterReceiver<T> = tokio::sync::mpsc::Receiver<T>;
+
+/// Create router channel
+pub fn router_channel<T>() -> (RouterSender<T>, RouterReceiver<T>) {
+    tokio::sync::mpsc::channel(64)
+}
+
+// TODO: Consider replacing with oneshot
```

Comment:
I _really_ like Oneshot! 

We're starting to see specialization of channel types so it works to start naming them by function (like e.g. MessageSender, MessageReceiver).

This also relates a little to @spacekookie's recent renaming for detached contexts and your use of a `Mailbox` abstraction in your access control PR. 

Basically, anything we can do that moves us further along the lines of generalizing the many different situations where we're using `mpsc` queues to communicate between ockam subsystems into a shared channel abstraction with common logic is a massive win IMO.

Erlang here we come!